src/shared/api/cbioportalInternalClientInstance.ts

```diff
@@ -1,5 +1,28 @@
 import { CBioPortalAPIInternal } from 'cbioportal-ts-api-client';
+import { getLoadConfig } from 'config/config';
+import { getBrowserWindow } from 'cbioportal-frontend-commons';
+
+function proxyColumnStore(client: any, endpoint: string) {
+    if (getBrowserWindow().location.search.includes('legacy')) {
+        return;
+    }
+
+    const method = `${endpoint}UsingPOSTWithHttpInfo`;
+    const old = client[method];
+    client[method] = function(params: any) {
+        params.$domain = `//${getLoadConfig().baseUrl}/api/column-store`;
+        const url = old.apply(this, [params]);
+        return url;
+    };
+}
 
 const internalClient = new CBioPortalAPIInternal();
 
+export const internalClientColumnStore = new CBioPortalAPIInternal();
+
+proxyColumnStore(internalClientColumnStore, 'fetchMutatedGenes');

```

Comment:
@haynescd right here is the ONLY thing you need to change to add an endpoint.  the string obviously corresponds to a fetch method on the client which corresponds to an endpoint